toucan_connectors/hubspot_private_app/hubspot_connector.py

```diff
@@ -103,6 +103,9 @@ def _result_iterator(
     ) -> Generator[_HubSpotResult, None, None]:
         after = None
         count = 0
+        # HubSpot returns a 400 HTTP error when trying to fetch 100 results
+        if limit is not None:
+            limit = min(limit, 100)
```

Comment:
then is it not better to have :
```suggestion
            limit = min(limit, 99)
```

Since 100results -> cause a 400 error...

just by curiosity... what's the message in the response for that 400 error... please ? 🤔  Is it a know issue or... ?

Reply:
The message is "You cannot fetch more than 100 results"

Reply:
adapted the comment to be clearer